bigtable/client/table.h

```diff
@@ -28,27 +28,6 @@
 
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
-class DataClient {
- public:
-  virtual ~DataClient() = default;
-
-  virtual std::string const& ProjectId() const = 0;
-  virtual std::string const& InstanceId() const = 0;
-
-  // Access the stub to send RPC calls.
-  virtual google::bigtable::v2::Bigtable::StubInterface& Stub() const = 0;
-};
-
-/// Create the default implementation of ClientInterface.
-std::shared_ptr<DataClient> CreateDefaultClient(std::string project_id,
-                                                std::string instance_id,
-                                                ClientOptions options);
-
-inline std::string InstanceName(std::shared_ptr<DataClient> client) {
-  return absl::StrCat("projects/", client->ProjectId(), "/instances/",
-                      client->InstanceId());
-}
-
 inline std::string TableName(std::shared_ptr<DataClient> client,
                              absl::string_view table_id) {
   return absl::StrCat(InstanceName(std::move(client)), "/tables/", table_id);
```

Comment:
Can you please add a header comment to class Table. (Not here, but two lines down.) The one you did for DataClient is helpful.

Reply:
Done.